### ISSUE TITLE
dramatic increase in score

### ISSUE THREAD

**Issue by ModestSkillsCH** [Tornith/CHSplit#3]
**Describe the bug**
I was playing kickapoo by tenacious D and on the outro somehow i got +1 billion points so my total was 1,065,353,216 when ch says 502,174

**To Reproduce**
dont know how to yet, just a heads up on this bug

**Expected behavior**
Expected CHSplit to split my score correctly and have the total score the same as clone hero

**Screenshots**
https://drive.google.com/file/d/12FR7Ifev1nk7CAJ54HeGfq0fOttq1HfC/view?usp=sharing

**Desktop (please complete the following information):**
 - OS: Windows 10
 - CPU: Intel I5-8250U
 - Clone Hero version: 23.2.2
 - CHSplit Version: 0.2
 - Any modifications to Clone Hero: No Mods

**Additional context**
No other context

**Comment by Tornith** [comment]
Oh this is interesting lmao, I'll look into what possibly could have gone wrong but this could be anything.

**Comment by ModestSkillsCH** [comment]
One possibility i thought of, was that somehow it stored up all my scores
from previous runs and added them together but even then i dont think it
would have added up to 1 billion

**Comment by Tornith** [comment]
No, that wouldn't be the case, my guess is that the song somehow ended way to quickly for CHSplit and it then read the memory incorrectly spewing random memory data and interpreting it as an integer.

**Comment by gmpdevabr** [comment]
not random though, got the same problem with a score of 1,065,353,216 https://prnt.sc/r6imuf
